src/live.asp.net/Services/FeedService.cs

```diff
@@ -78,86 +78,73 @@ public AtomFeed()
         [XmlNamespaceDeclarations]
         public XmlSerializerNamespaces Namespaces { get; set; }
 
-        [XmlElement("id")]
-        public string Id { get; set; }
+        public string id { get; set; }
```

Comment:
I know what you gonna say, but always refer to [CONTRIBUTING.md](https://github.com/aspnet/live.asp.net/blob/dev/CONTRIBUTING.md)

BTW what you're doing is good as contributor but you should follow the guidance, I suggest to finish everything and squash the commits if it's possible, after you can wait for **Damian Edwards** approval because he is the man behind the entire stuff in live.asp.net